src/core_plugins/vega/public/vega_view/vega_base_view.js

```diff
@@ -192,10 +191,8 @@ export class VegaBaseView {
    * Set global debug variable to simplify vega debugging in console. Show info message first time
    */
   setDebugValues(view, spec, vlspec) {
```

Comment:
We should remove this function (and it's calls completely). It's not having any use anymore, and we are using git if it's just for the sake of being able to look at the source code again :))